A_Getting_started/lesson1.rst

```diff
@@ -1,11 +1,11 @@
-Lesson 1: Creating a Standing Model
+Lesson 1: Creating a Human Standing Model
 ==================================================
 
-The standing model can be found in the AMMR folder under
-Applications/Examples. The main file is called StandingModel.Main.any.
+The Human Standing model can be found in the AMMR folder under
```

Comment:
Yes. Just revert the "Human Standing" thing.

Reply:
Done. Just committed new files.